js/wdi_radio/edit.directive.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+(function(){
+  angular
+  .module("wdiRadio")
+  .directive("editForm", [
```

Comment:
Since this form can really create, update, or delete a song, I think it might be more semantic to call this `form.directive.js` instead